geometry_msgs/Point q_near
geometry_msgs/Point q_new
---
bool valid
